src/python/pants/backend/python/goals/export.py

```diff
@@ -188,7 +191,10 @@ async def do_export(
         return ExportResult(
             description,
             dest,
-            post_processing_cmds=[PostProcessingCommand(["ln", "-s", venv_abspath, output_path])],
+            post_processing_cmds=[
+                PostProcessingCommand(["rmdir", output_path]),
```

Comment:
export creates an empty directory for us when the digest gets written. We have to remove that before creating the symlink in its place.

Reply:
This should be a comment in the code...

Reply:
Done